README.md

```diff
@@ -117,16 +117,21 @@ sh 1.run_preprocessing.sh
 ```
 **Slurm**:
 ```sh
-sh 1.run_preprocessing_slurm.sh
+sbatch 1.run_preprocessing_slurm.sh
 ```
 
+***Note**: For testing purposes, users may only have generated sbs or phenotype images.
+If either of the dataframes defining the file paths and metadata for sbs/phenotype samples (located at `SBS_SAMPLES_DF_FP`/`PHENOTYPE_SAMPLES_DF_FP`) are empty, then no preprocessing steps will be run for the respective missing data type.
+Use an empty dataframe for the sbs/phenotype samples to skip preprocessing for a particular data type.
```

Comment:
Ok, good to know. Can you also restrict this using an until flag?

Reply:
Not as of now. We could create two additional rules in `workflow/rules/preprocess.smk` for the following targets:
1) PREPROCESS_TARGETS_SBS
2) PREPROCESS_TARGETS_PHENOTYPE

Then one should be able to restrict preprocessing using an until flag. Should I add these two rules?

Reply:
We can think about that if it makes sense down the road. If we can't use until flags for this, that's fine--don't add this into the readme then